services/ui-src/src/components/pages/ReviewSubmit/ReviewSubmitPage.tsx

```diff
@@ -154,7 +154,7 @@ const ReadyToSubmit = ({
         </Heading>
         <Box sx={sx.infoTextBox}>
           <Text sx={sx.infoHeading}>{intro.infoHeader}</Text>
-          <Text>{parseCustomHtml(intro.info)}</Text>
+          <Box>{parseCustomHtml(intro.info)}</Box>
```

Comment:
FYI this may conflict with the changes done in #490  And I think this doesn't really impact the Alert component, so maybe we can revert this particular change?

Reply:
Ok, I can revert this

Reply:
Or I can wait til your PR is in